.github/workflows/cbdb_release_weekly.yml

```diff
@@ -0,0 +1,80 @@
+name: Build and Release RPM Weeklyly
```

Comment:
Weeklyly -> Weekly

Reply:
Thanks for your correction, I will modify it